README.md

```diff
@@ -38,8 +38,10 @@ In VSCode:
 
 4. Navigate to the downloaded release file (step 1) and select it.
 
-5. Select theme IES Highlighter dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Colour Theme.
+5. Select theme IES Highlighter <sup>1</sup>dark+ default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Colour Theme.
```

Comment:
Generally these are placed after the text but maybe you didn't want to mix it with the plus.
```suggestion
5. Select theme IES Highlighter dark+<sup>1</sup> default `Cmd/Ctrl + k` then `Cmd/Ctrl + t`. Or go to the Extensions view -> ies-extension -> Set Colour Theme.
```

Reply:
The back and forth is getting inefficient - lets just merge this, and let you open a PR and drive the changes?